openedx/core/djangoapps/models/course_details.py

```diff
@@ -258,7 +259,8 @@ def update_from_json(cls, course_key, jsondict, user):  # pylint: disable=too-ma
             dirty = True
 
         if 'instructor_info' in jsondict:
-            descriptor.instructor_info = jsondict['instructor_info']
+            instructor_info = cls._check_for_unique_id(jsondict['instructor_info'])
```

Comment:
@douglashall, Browser/Javascript has no access to MAC address, so we cannot generate the UUIDs in javascript(we can generate random ids but they are not guaranteed to be unique). That is why we are doing this in python.